lib/core/configurator.c

```diff
@@ -240,6 +240,50 @@ static int sort_from_longer_paths(const yoml_mapping_element_t *x, const yoml_ma
     return strcmp(x->key->data.scalar, y->key->data.scalar);
 }
 
+static yoml_t *convert_path_config_node(h2o_configurator_command_t *cmd, yoml_t *node)
+{
+    size_t i, j;
+
+    switch (node->type) {
+    case YOML_TYPE_MAPPING:
+        break;
+    case YOML_TYPE_SEQUENCE: {
+        /* convert to mapping */
+        yoml_t *map = h2o_mem_alloc(offsetof(yoml_t, data.mapping.elements));
+        map->type = YOML_TYPE_MAPPING;
+        map->_refcnt = 1;
+        map->data.mapping.size = 0;
```

Comment:
You should never leave some properties (e.g. `anchor`) uninitialized. This would lead to undefined behavior when the value is used (e.g. in `yoml_free`). And regarding _what_ to set, it might be a good idea to inherit `filename`, `line`, and `column` from the original node.